code/_onclick/hud/new_player.dm

```diff
@@ -256,13 +267,24 @@
 	set_button_status(TRUE)
 	UnregisterSignal(SSticker, COMSIG_TICKER_ENTER_PREGAME)
 
-/atom/movable/screen/lobby/button/settings
+//Subtype the bottom buttons away so the collapse/expand shutter goes behind them
+/atom/movable/screen/lobby/button/bottom
+	layer = LOBBY_BOTTOM_BUTTON_LAYER
 	icon = 'icons/hud/lobby/bottom_buttons.dmi'
+	always_shown = TRUE
+	var/matrix/button_matrix = matrix()
+
+/atom/movable/screen/lobby/button/bottom/SlowInit()
+	. = ..()
```

Comment:
why is this in SlowInit?

Reply:
https://github.com/tgstation/tgstation/commit/28ffee160e7e3ecd6602535f6400597692c0b938